tests/test_SklearnGLMClassifierConverter.py

```diff
@@ -11,6 +11,12 @@
 from test_utils import dump_data_and_model
 
 
+def sklearn_version():
```

Comment:
Do you want to name this _sklearn_version() as it's an internal function?

Reply:
Fixed.